src/codecs/encoding/transformer.rs

```diff
@@ -276,7 +265,7 @@ mod tests {
     #[test]
     fn deserialize_and_transform_only() {
         let transformer: Transformer =
-            toml::from_str(r#"only_fields = ["a.b.c", "b", "c[0].y", "g\\.z"]"#).unwrap();
+            toml::from_str(r#"only_fields = ["a.b.c", "b", "c[0].y", "\"g.z\""]"#).unwrap();
```

Comment:
I'm trying to understand if this change has any implication for backward compatibility?

Reply:
This was supposed to be using the "new" syntax already, but the test was broken (but still passed). It is now fixed.